### PR TITLE
chore: update the CDK Bootstrap template to the latest version

### PR DESCRIPTION
**Pull request by philasmar**
*Description of changes:*
update the CDK Bootstrap template to the latest version

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by ashovlin — APPROVED** [review]

File: src/AWS.Deploy.Orchestration/CDK/CDKBootstrapTemplate.yaml [hunk 1]
```diff
@@ -489,7 +490,7 @@ Resources:
       Type: String
       Name:
         Fn::Sub: /cdk-bootstrap/${Qualifier}/version
-      Value: "12"
+      Value: "13"
```
Comment (normj) [inline src/AWS.Deploy.Orchestration/CDK/CDKBootstrapTemplate.yaml:493]: You need to also update the [Constant]( https://github.com/aws/aws-dotnet-deploy/blob/5b78ecaec217908c19d92fabbdfcf1963821194a/src/AWS.Deploy.Constants/CDK.cs#L19)

If you are filling ambitious you get rid of the constant and parse the version from the bootstrap yaml file so we don't have to remember to update both places in the future.

**Review by normj — CHANGES_REQUESTED** [review]

File: src/AWS.Deploy.Orchestration/CDK/CDKBootstrapTemplate.yaml (hunk 1 above)
Comment (philasmar) [reply src/AWS.Deploy.Orchestration/CDK/CDKBootstrapTemplate.yaml:493]: Done

**Review by 96malhar — CHANGES_REQUESTED** [review]
Can we add a unit test for ` ReadCDKTemplateVersion` ?

**Comment by 96malhar** [comment]
> Can we add a unit test for ` ReadCDKTemplateVersion` ?

Synced up with @philasmar offline and verified that the new method is being covered by an existing test - https://github.com/aws/aws-dotnet-deploy/blob/main/test/AWS.Deploy.Orchestration.UnitTests/CDK/CDKProjectHandlerTests.cs#L126

**Review by 96malhar — APPROVED** [review]

**Pull request merged**